Core/HLE/sceDisplay.cpp

```diff
@@ -106,7 +106,7 @@ static int height;
 static bool wasPaused;
 static bool flippedThisFrame;
 
-static int framerate = 60;
+static int framerate = g_Config.iDisplayRefreshRate;
```

Comment:
This initialization should probably goes to `void __DisplayInit() `, including the one below it since it relies on this variable)

Reply:
No this is fine, the emulator doesn't have any issues and tests pass as-is. Just as before this PR.

Unless you mean to reduce code duplication by calling `__DisplaySetFramerate()` in `__DisplayInit()`.

Reply:
Because `__DisplaySetFramerate()` now checks for VR, and Force72Hz is on by default, the tests could fail if they somehow run in VR mode since that'll speed things up.

Doesn't seem necessary, and if anything there's probably one frame of latency before the VR's 72 Hz value gets engaged. No way a human could ever notice :)

Reply:
i'm not sure whether the `g_Config.iDisplayRefreshRate` here already contained the value loaded from PPSSPP.ini or not probably not yet.

Reply:
Indeed, the user configuration will probably load in the second frame. Not noticeable by humans, but I'll do it if you wish.

Reply:
Well, the proper way to use `g_Config` is to wait until it's loaded from PPSSPP.ini before using it to set/calculate something.